internal/context.go

```diff
@@ -51,6 +52,12 @@ func SetContext(contextName, comment string) string {
 		LogVerbose("Creating context: " + contextName)
 	}
 
+	// Check if certificates should be added, if so we should add them with an absolute path
+	certificates := viper.GetString("certificates")
+	if certificates != "" {
+		certificates, _ = filepath.Abs(certificates)
```

Comment:
Is this needed? Should already be an absolute path in viper.

Reply:
True! Good catch. Fixed it now.